spec/namespaces/flow_framework.yaml

```diff
@@ -0,0 +1,294 @@
+openapi: 3.1.0
+info:
+  title: OpenSearch Flow Framework API
+  description: OpenSearch Flow Framework API.
+  version: 1.0.0
+paths:
+  /_plugins/_flow_framework/workflow:
+    post:
+      operationId: flow_framework.create.0
+      x-operation-group: flow_framework.create
+      x-version-added: '2.12'
+      description: Create a workflow.
+      externalDocs:
+        url: https://opensearch.org/docs/latest/automating-configurations/api/create-workflow/
+      parameters:
+        - $ref: '#/components/parameters/flow_framework.create::query.provision'
+        - $ref: '#/components/parameters/flow_framework.create::query.validation'
+        # eslint-disable-next-line @cspell/spellchecker
+        - $ref: '#/components/parameters/flow_framework.create::query.reprovision'
```

Comment:
I still think you're missing `use_case`: https://opensearch.org/docs/latest/automating-configurations/workflow-templates/